ftplugin/php/splitjoin.vim

```diff
@@ -1,6 +1,6 @@
 let b:splitjoin_split_callbacks = [
       \ 'sj#js#SplitArray',
-      \ 'sj#php#SplitArray',
+      \ 'sj#php#SplitBraces',
```

Comment:
Dao questions :smile: Please do it the way you think it needs to happen. When I want to split / join I explicitly put cursor where thats needed, I don't expect it'll magically guess my intention. But it seems thats not your plan so please add a change so it would adhere to your desired behavior.